website/views.py

```diff
@@ -235,6 +235,15 @@ def dashboard(auth):
             'dashboard_id': dashboard_id,
             }
 
+#TODO
+@must_be_logged_in
+def preregAdmin(auth):
+    user = auth.user
+    dashboard_folder = find_dashboard(user)
+    dashboard_id = dashboard_folder._id
+    return {'addons_enabled': user.get_addon_names(),
```

Comment:
@lyndsysimon this code isn't used anymore and will be removed so you can ignore the preregAdmin stuff